src/components/Header/Header.js

```diff
@@ -1,37 +1,42 @@
 import React from 'react';
-import PropTypes from 'prop-types';
-import Link from '../Link/Link';
 import {
   HeaderWrapper,
   LogoContainer,
   LinksContainer,
+  NavLink,
 } from './Header.style';
+import Logo from '../../assets/images/logo.svg';
 
-function Header({ logo, links }) {
+const headerLinks = [
+  {
+    title: 'Search',
+    link: '/search?title="javascript"',
+  },
+  {
+    title: 'How it works',
+    link: '/#how-it-works',
+  },
+  {
+    title: 'About',
+    link: '/#about',
+  },
+];
+
+function Header() {
```

Comment:
I had a look in the browser now and saw that you can simplify the header quite a bit. Currently the header looks like this

```
<header>
  <div>
    <a alt="" href="/">
      <img src="/static/media/logo.1528c9cf.svg" alt="Logo">
    </a>
  </div>
  <nav>
    <a alt="Search" href="/search?title=&quot;javascript&quot;">Search</a>
    <a alt="How it works" href="/#how-it-works">How it works</a>
    <a alt="About" href="/#about">About</a>
  </nav>
</header>
```

You could simplify it to be like this

```
<header>
  <a alt="" href="/">
    <img src="/static/media/logo.1528c9cf.svg" alt="Logo">
  </a>

  <nav>
    <a alt="Search" href="/search?title=&quot;javascript&quot;">Search</a>
    <a alt="How it works" href="/#how-it-works">How it works</a>
    <a alt="About" href="/#about">About</a>
  </nav>
</header>
```

- <header> should have padding: 0 80px, display: flex, align-items: center, justify-content: space-between => would get rid of the LogoContainer
- <nav> doesn't need align-items, justify-content and padding
- the logo link should not be a NavLink since its styling needs to be different

This way you simplified the element a bit, but especially the CSS will benefit